src/sentry/static/sentry/app/views/organizationIntegrations/integrationServerlessRow.tsx

```diff
@@ -83,20 +83,35 @@ class IntegrationServerlessRow extends React.Component<Props> {
       addErrorMessage(t('An error ocurred'));
     }
   };
+  renderLayerStatus() {
+    const {serverlessFunction} = this.props;
+    if (serverlessFunction.outOfDate) {
+      return (
+        <UpdateButton size="small" priority="primary" onClick={this.updateVersion}>
+          {t('Update')}
+        </UpdateButton>
+      );
+    }
+    return this.enabled ? t('Latest') : t('Disabled');
```

Comment:
total nit: But I would swap these conditionals, minorly easier to read when the if condition is just a one line return

Reply:
eh, I think this is easier to read in multiple lines

Reply:
```jsx
  renderLayerStatus() {
    const {serverlessFunction} = this.props;
    if (!serverlessFunction.outOfDate) {
		return this.enabled ? t('Latest') : t('Disabled');
	}

    return (
      <UpdateButton size="small" priority="primary" onClick={this.updateVersion}>
        {t('Update')}
      </UpdateButton>
    );
}
```

Reply:
Is what I meant, less pyramid-y

Reply:
oh got it, yea

Reply:
Like I said tho, totally a nit lol

Reply:
I have a test failing so I change this as well :)